server.js

```diff
@@ -22,6 +22,19 @@ const {actions, audit} = require('./bot.js');
 
 app.use(express.json());
 
+const { createProbot } = require("probot");
+const approvals = require("./approvals/app");
+const probot = createProbot();
+
+app.post('/approvals', async function(request, response) {
+  await probot.load(approvals);
+  probot.webhooks.receive({
+    name: request.headers['x-github-event'],
+    payload: request.body
+  });
+  response.send(200);
```

Comment:
Since we're running it on a glitch right now, which have no meaningful store for logs anyways, I'd say for now it is an overkill.

Reply:
I think the idea is that the non-200 responses show up with the error in the body in a GitHub Actions dashboard, based on the info in that blog post. So it could help with debugging even though Glitch doesn't expose the logs.

But I've never tried that out, and we don't need to block the PR on that.

Reply:
added an issue for now: https://github.com/GoogleChromeLabs/chrome-devrel-review-bot/issues/13